daemon/logger/copier.go

```diff
@@ -44,8 +43,7 @@ func (c *Copier) copySrc(name string, src io.Reader) {
 	reader := bufio.NewReader(src)
 
 	for {
-		line, err := reader.ReadBytes('\n')
-		line = bytes.TrimSuffix(line, []byte{'\n'})
+		line, _, err := reader.ReadLine()
```

Comment:
@chenchun we might as well not do `bufio.NewReader(src)`, and simply call `src.Read(p)` with a specified buffer size for `p`. WDYT?